medusa/name_parser/parser.py

```diff
@@ -359,6 +356,17 @@ def __str__(self):
                                              quality=common.Quality.qualityStrings[self.quality]))
         return helpers.canonical_name(obj, fmt='{key}: {value}', separator=', ')
 
+    def get_quality(self, guess, extend=False):
+        """Return video quality from guess or name.
+
+        :return:
+        :rtype: Quality
+        """
+        quality = common.Quality.from_guessit(guess)
+        if quality != common.Quality.UNKNOWN:
+            return quality
+        return common.Quality.nameQuality(self.original_name, self.is_anime, extend)
```

Comment:
Good question. They should be: https://github.com/pymedusa/Medusa/blob/2697d373701d7bea6dcfe6a651e3d2ff0a23df80/medusa/name_parser/parser.py#L315-L319

self.original_name and self.ab_episode_numbers